internal/pkg/aws/ec2/ec2.go

```diff
@@ -101,12 +128,19 @@ func (c *EC2) ListVPC() ([]string, error) {
 		}
 		vpcs = append(vpcs, response.Vpcs...)
 	}
-	var vpcNames []string
+	var VPCs []VPC
```

Comment:
it's a bit odd to have allcaps local variable names (https://golang.org/doc/effective_go.html#mixed-caps)
```go
	var ec2VPCs 	[]*ec2.Vpc
	// blah blah blah   

	var vpcs []VPC // our vpc definition
```